custom_formatter.rb

```diff
@@ -1,11 +0,0 @@
-require "rspec/core/formatters/base_text_formatter"
```

Comment:
Ahh, OK. Makes sense. To be honest, I haven't mucked with this in a while and have forgotten a bit :smile_cat: